polyfills/navigator/sendBeacon/polyfill.js

```diff
@@ -0,0 +1,33 @@
+(function(root) {
+
+	function sendBeacon(url, data) {
+		var xhr = ('XMLHttpRequest' in window) ? new XMLHttpRequest() : new ActiveXObject('Microsoft.XMLHTTP');
+		xhr.open('POST', url, false);
+		xhr.setRequestHeader('Accept', '*/*');
+		if (typeof data === 'string') {
+			xhr.setRequestHeader('Content-Type', 'text/plain;charset=UTF-8');
+			xhr.responseType = 'text/plain';
+		} else if (Object.prototype.toString.call(data) === '[object Blob]') {
+			if (data.type) {
+				xhr.setRequestHeader('Content-Type', data.type);
+			}
+		}
+		xhr.send(data);
+		return true;
+	}
+
+	if (!('sendBeacon' in navigator)) {
+		navigator.sendBeacon = sendBeacon;
```

Comment:
It's not doing any harm, and I'd rather keep the polyfill aligned with the third party source.  I'll push the navigator change upstream.